test/services/snapshot-service.test.ts

```diff
@@ -37,6 +39,21 @@ describe('SnapshotService', () => {
       expect(snapshotResponse.body).to.deep.equal({data: {id: snapshotId}})
       expect(snapshotResponse.statusCode).to.eq(200)
     })
+
+    it('creates multiple snapshots', async () => {
```

Comment:
I can confirm on master, this test passes when `length: 1` and fails with `length: 2` or more.